docs/tutorial.md

```diff
@@ -0,0 +1,103 @@
+- My Community Currency contract address is [0x8611c307F3b88040Aa4E73E8e2c5DB303ca81701](https://ropsten.etherscan.io/address/0x8611c307f3b88040aa4e73e8e2c5db303ca81701), save address of your Community Currency cause we're going to use it soon.
+
+So what happened here?
+
+- `CurrencyFactory` created the CC. As the token creator the `CurrencyFactory` contract was made the owner of all the suply.
+- `CurrencyFactory` created the `MarketMaker`, that's the unkown contract we saw! `MarketMaker` is used to exchange CLN/CC. More about it latter.
+- `CurrencyFactory` moved all CC suply to the `MarketMaker`. As an exchange provider it's reasonable that the MM will hold all the CC.
+
+Nice, we created a currency but all of it is locked in some `MarketMaker`. We should send CLN to get us some of that crypto-dough. But if we just send CLN to the `MarketMaker` he will not have a slicest idea what we want. It will be hard to get this money back, so **don't** do this. Instead, there is two functions on the `CurrencyFactory` that designed exactly for that - `insertCLNtoMarketMaker` and `extractCLNfromMarketMaker`, we need the former.
+
+Before calling this function I need to approve `CurrencyFactory` to use my CLN tokens. It's like saying to CLN contract: "Hey contract, if someone named CurrencyFactory will try to use my tokens, it's ok, I approve this". This is a ERC20 mechanism.
+
+ Remeber how we called `createIssuance`? Now we'll do the same with the `ColuLocalNetwork` token. After uploading the ABI I'll call approve function with 1000 * 1e18 = 1000000000000000000000 (1000 CLN) as value and `CurrencyFactory` address as the spender. I check everything is correct and click send. Easy, the only thing to know about approve, that if you already approved some amount and you want to change it, you need first to set allowance  to zero and then send approve again with the new value ([more info](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md#approve )).
+
+ After the approve transaction is confirmed I can eventually call `insertCLNtoMarketMaker`. I do the same process again, this time for `CurrencyFactory`, after ABI is loaded I select `insertCLNtoMarketMaker` function. There's actually two functions with the same name (this is Solidity's function overloading feature), and I need the one that received both `token` and `clnAmount`. Maybe we talk latter about the second one. In the token field I paste the `CommunityCurrency`'s address, and put the same 1000 * 1e18 for the ammount field.
+
+Viewing the [transaction](https://ropsten.etherscan.io/tx/0x350fe7bad490baa8a0446c8f5f76bb913b8238fcd882832bb7b4b3e354d1b9c6) you may think some complex stuff happened there. Well, I'll try to sum it up.
+
+- `CurencyFactory` transfers the CLN to itself. He can do it cause I gave him an allowance for this before.
```

Comment:
* CurencyFactory --> CurrencyFactory
* He can do it cause I gave him --> It can do it cause I gave it